internal/exp/metrics/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [sh0rez]
```

Comment:
Isn't this the source of the codeowners file?

Reply:
the autogen does not work for me (not an org member, so the script cannot fetch the member list). according to contributing.md you have do edit manually then. 

updated the yaml to reflect the codeowners file